### ISSUE TITLE
Only on Wifi

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#34]
```
Can you add an option that only make the app only use Wifi?
I don't want the app wasting my data considering there is no time I'll be using 
the app without Wifi.
```

Original issue reported on code.google.com by `marcus1...@gmail.com` on 7 Jan 2011 at 2:31

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 7 Jan 2011 at 9:15
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Or when USB plugged, that will be cool too. But only if we don't need to 
install and setup the SDK. (Don't know if it's possible)
```

Original comment by `Guillaum...@gmail.com` on 7 Jan 2011 at 3:25

**Comment by GoogleCodeExporter** [comment]
```
> Or when USB plugged, that will be cool too.

GTalkSMS already has the ability to auto connect when charging and disconnect 
when not charging.

FWIW, much finer control over things can be achieved by using the "Tasker" 
application - see the wiki for instructions on how to integrate with that.
```

Original comment by `skippy.hammond@gmail.com` on 7 Jan 2011 at 11:44

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 8 Jan 2011 at 12:56
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I would like to have a new autostart option: on wifi connect!
Is this possible?
```

Original comment by `Matt.f...@gmail.com` on 22 Jan 2011 at 9:23

**Comment by GoogleCodeExporter** [comment]
```
This feature is in the pipe.
I don't know when I will start it. Probably in few weeks.
```

Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 10:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 29 Jan 2011 at 7:02
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 29 Jan 2011 at 7:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 29 Jan 2011 at 9:31
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.8
```

Original comment by `Florent....@gmail.com` on 29 Jan 2011 at 10:57
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
